queries.md

```diff
@@ -5,75 +5,93 @@
-
 <!-- Your Code Goes Here -->
+Filter {"acquisition.price_amount": {$gt:100000000}, founded_year:{$lt:2010}} 
+Project {ipo:1,name:1,_id:0}
 
 ### 5. All the companies that have less than 1000 employees and have been founded before 2005. Order them by the number of employees and limit the search to 10 companies.
-
 <!-- Your Code Goes Here -->
+Filter {number_of_employees: {$lte:1000}, founded_year:{$gte:2005}}
+Sort {number_of_employees:1}
+Limit 10
 
 ### 6. All the companies that don't include the `partners` field.
-
 <!-- Your Code Goes Here -->
+Filter {partners: {$exists:false}}
 
 ### 7. All the companies that have a null type of value on the `category_code` field.
-
 <!-- Your Code Goes Here -->
+Filter {category_code:"null"}
 
 ### 8. All the companies that have at least 100 employees but less than 1000. Retrieve only the `name` and `number of employees` fields.
-
 <!-- Your Code Goes Here -->
+Filter {number_of_employees: {$lte:1000,$gte:1000}}
+Project {name:1, number_of_employees:1,  _id:0}
 
 ### 9. Order all the companies by their IPO price in a descending order.
-
 <!-- Your Code Goes Here -->
+Sort {ipo:-1}
 
 ### 10. Retrieve the 10 companies with most employees, order by the `number of employees`
-
 <!-- Your Code Goes Here -->
+Sort {number_of_employees:-1}
+Limit 10
 
 ### 11. All the companies founded on the second semester of the year. Limit your search to 1000 companies.
-
 <!-- Your Code Goes Here -->
+Filter {founded_month:{$gte:6}}
+Limit 1000
 
 ### 12. All the companies founded before 2000 that have an acquisition amount of more than 10.000.000
-
 <!-- Your Code Goes Here -->
+Filter {founded_year: {$lte: 2000}, "acquisition.price_amount": {$gt:10000000}}
 
 ### 13. All the companies that have been acquired after 2010, order by the acquisition amount, and retrieve only their `name` and `acquisition` field.
-
 <!-- Your Code Goes Here -->
+Filter {"acquisition.acquired_year":{$gt:2010}}
+Project {name:1, acquisition:1, _id:0}
+Sort {"acquisition.price_amount":-1}
 
 ### 14. Order the companies by their `founded year`, retrieving only their `name` and `founded year`.
-
 <!-- Your Code Goes Here -->
+Project {name:1,founded_day:1,_id:0}
+Sort {founded_year:-1}
 
 ### 15. All the companies that have been founded on the first seven days of the month, including the seventh. Sort them by their `acquisition price` in a descending order. Limit the search to 10 documents.
-
 <!-- Your Code Goes Here -->
+Filter {founded_day: {$lte:7}}
+Sort {"acquisition.price_amount":-1}
+Limit 10
 
 ### 16. All the companies on the 'web' `category` that have more than 4000 employees. Sort them by the amount of employees in ascending order.
-
 <!-- Your Code Goes Here -->
+Filter {category_code: "web", number_of_employees:{$gt:400}}
+Sort {number_of_employees:1}
 
 ### 17. All the companies whose acquisition amount is more than 10.000.000, and currency is 'EUR'.
-
```

Comment:
te paso la respuesta para que le eches un ojito: 
query:{$and:[{"acquisition.price_amount": {$gt: 10000000}},{"acquisition.price_currency_code": "EUR"}]}